TGEngine/io/Resource.cpp

```diff
@@ -39,17 +40,62 @@ namespace tge::io {
 		uint32_t lastVertexCount = 0;
 		uint32_t lastIndexCount = 0;
 
+		uint32_t instanceCount = 0;
+
+		std::vector<BufferObject> instanceStaging;
+		std::vector<VkBufferCopy> instanceStagingCopy;
+		instanceStaging.reserve(actorCount); // Speculative
+		instanceStagingCopy.reserve(actorCount); // Speculative
+
 		// Goes on until it hits a change request block size (2^32)
 		for (uint32_t currentId = 0; currentId < actorCount; currentId++) {
 
 			// Reads the actor properties and so on
 			ActorProperties* currentProperty = actorProperties.data() + currentId;
 			ActorDescriptor* currentDescription = actorDescriptor.data() + currentId;
 
+			uint32_t cinstanceCount = 0;
```

Comment:
`cinstanceCount` is pretty ambiguous, what does the c stand for? current?
anyway it should also follow camelcase standards

Reply:
Yes c stands for current. I will change it tho